web-animations-1/Overview.bs

```diff
@@ -4699,6 +4711,18 @@ interface KeyframeEffect : AnimationEffect {
 
     1.  Set the <a>target element</a> of <var>effect</var> to <var>target</var>.
 
+    1.  Set the <a>target pseudo-selector</a> of <var>effect</var> to the result
+        corresponding to the first matching condition from below.
+
+        :   If <var>options</var> is a {{KeyframeEffectOptions}} object with a
+            {{KeyframeEffectOptions/pseudoElement}} property,
+        ::  Set the [=target pseudo-selector=] to the value of that property,
+            following the validation procedure from the
+            {{KeyframeEffect/pseudoElement}} setter.
```

Comment:
Nit: For consistency, could we follow the same format we use for `composite`?

e.g.

```bs
Set the [=target pseudo-selector=] to the value of the {{KeyframeEffectOptions/pseudoElement}} property.

When assigning this property, the error-handling defined for the corresponding setter on the
{{KeyframeEffect}} interface is applied. If the setter requires an exception to be thrown, this procedure
must throw the same exception and abort all further steps.
```

Reply:
done